// @openzeppelin/contracts-upgradeable/build/contracts/CrossChainEnabledPolygonChildUpgradeable__factory.ts
/* Autogenerated file. Do not edit manually. */
/* tslint:disable */
/* eslint-disable */

import { Contract, Signer, utils } from "ethers";
import type { Provider } from "@ethersproject/providers";
import type {
  CrossChainEnabledPolygonChildUpgradeable,
  CrossChainEnabledPolygonChildUpgradeableInterface,
} from "../../../../../../node_modules/@openzeppelin/contracts-upgradeable/build/contracts/CrossChainEnabledPolygonChildUpgradeable";

const _abi = [
  {
    anonymous: false,
    inputs: [
      {
        indexed: false,
        internalType: "uint8",
        name: "version",
        type: "uint8",
      },
    ],
    name: "Initialized",
    type: "event",
  },
  {
    inputs: [
      {
        internalType: "uint256",
        name: "",
        type: "uint256",
      },
      {
        internalType: "address",
        name: "rootMessageSender",
        type: "address",
      },
      {
        internalType: "bytes",
        name: "data",
        type: "bytes",
      },
    ],
    name: "processMessageFromRoot",
    outputs: [],
    stateMutability: "nonpayable",
    type: "function",
  },
];

export class CrossChainEnabledPolygonChildUpgradeable__factory {
  static readonly abi = _abi;
  static createInterface(): CrossChainEnabledPolygonChildUpgradeableInterface {
    return new utils.Interface(
      _abi
    ) as CrossChainEnabledPolygonChildUpgradeableInterface;
  }
  static connect(
    address: string,
    signerOrProvider: Signer | Provider
  ): CrossChainEnabledPolygonChildUpgradeable {
    return new Contract(
      address,
      _abi,
      signerOrProvider
    ) as CrossChainEnabledPolygonChildUpgradeable;
  }
}
