// @openzeppelin/contracts-upgradeable/build/contracts/CrossChainEnabledAMBUpgradeable__factory.ts
/* Autogenerated file. Do not edit manually. */
/* tslint:disable */
/* eslint-disable */
import { Signer, utils, Contract, ContractFactory, Overrides } from "ethers";
import type { Provider, TransactionRequest } from "@ethersproject/providers";
import type {
  CrossChainEnabledAMBUpgradeable,
  CrossChainEnabledAMBUpgradeableInterface,
} from "../../../../../../node_modules/@openzeppelin/contracts-upgradeable/build/contracts/CrossChainEnabledAMBUpgradeable";

const _abi = [
  {
    inputs: [
      {
        internalType: "address",
        name: "bridge",
        type: "address",
      },
    ],
    stateMutability: "nonpayable",
    type: "constructor",
  },
  {
    anonymous: false,
    inputs: [
      {
        indexed: false,
        internalType: "uint8",
        name: "version",
        type: "uint8",
      },
    ],
    name: "Initialized",
    type: "event",
  },
];

const _bytecode =
  "0x60a0604052348015600f57600080fd5b5060405160bc38038060bc833981016040819052602a91603a565b6001600160a01b03166080526068565b600060208284031215604b57600080fd5b81516001600160a01b0381168114606157600080fd5b9392505050565b608051603f607d60003960005050603f6000f3fe6080604052600080fdfea2646970667358221220561016127b8883ae2bb772664c1b03742bee5272a341067e1f043e241a00654864736f6c63430008090033";

type CrossChainEnabledAMBUpgradeableConstructorParams =
  | [signer?: Signer]
  | ConstructorParameters<typeof ContractFactory>;

const isSuperArgs = (
  xs: CrossChainEnabledAMBUpgradeableConstructorParams
): xs is ConstructorParameters<typeof ContractFactory> => xs.length > 1;

export class CrossChainEnabledAMBUpgradeable__factory extends ContractFactory {
  constructor(...args: CrossChainEnabledAMBUpgradeableConstructorParams) {
    if (isSuperArgs(args)) {
      super(...args);
    } else {
      super(_abi, _bytecode, args[0]);
    }
  }

  override deploy(
    bridge: string,
    overrides?: Overrides & { from?: string | Promise<string> }
  ): Promise<CrossChainEnabledAMBUpgradeable> {
    return super.deploy(
      bridge,
      overrides || {}
    ) as Promise<CrossChainEnabledAMBUpgradeable>;
  }
  override getDeployTransaction(
    bridge: string,
    overrides?: Overrides & { from?: string | Promise<string> }
  ): TransactionRequest {
    return super.getDeployTransaction(bridge, overrides || {});
  }
  override attach(address: string): CrossChainEnabledAMBUpgradeable {
    return super.attach(address) as CrossChainEnabledAMBUpgradeable;
  }
  override connect(signer: Signer): CrossChainEnabledAMBUpgradeable__factory {
    return super.connect(signer) as CrossChainEnabledAMBUpgradeable__factory;
  }

  static readonly bytecode = _bytecode;
  static readonly abi = _abi;
  static createInterface(): CrossChainEnabledAMBUpgradeableInterface {
    return new utils.Interface(
      _abi
    ) as CrossChainEnabledAMBUpgradeableInterface;
  }
  static connect(
    address: string,
    signerOrProvider: Signer | Provider
  ): CrossChainEnabledAMBUpgradeable {
    return new Contract(
      address,
      _abi,
      signerOrProvider
    ) as CrossChainEnabledAMBUpgradeable;
  }
}
