// @openzeppelin/contracts-upgradeable/build/contracts/CrossChainEnabledOptimismUpgradeable__factory.ts
/* Autogenerated file. Do not edit manually. */
/* tslint:disable */
/* eslint-disable */

import { Contract, Signer, utils } from "ethers";
import type { Provider } from "@ethersproject/providers";
import type {
  CrossChainEnabledOptimismUpgradeable,
  CrossChainEnabledOptimismUpgradeableInterface,
} from "../../../../../../node_modules/@openzeppelin/contracts-upgradeable/build/contracts/CrossChainEnabledOptimismUpgradeable";

const _abi = [
  {
    anonymous: false,
    inputs: [
      {
        indexed: false,
        internalType: "uint8",
        name: "version",
        type: "uint8",
      },
    ],
    name: "Initialized",
    type: "event",
  },
];

export class CrossChainEnabledOptimismUpgradeable__factory {
  static readonly abi = _abi;
  static createInterface(): CrossChainEnabledOptimismUpgradeableInterface {
    return new utils.Interface(
      _abi
    ) as CrossChainEnabledOptimismUpgradeableInterface;
  }
  static connect(
    address: string,
    signerOrProvider: Signer | Provider
  ): CrossChainEnabledOptimismUpgradeable {
    return new Contract(
      address,
      _abi,
      signerOrProvider
    ) as CrossChainEnabledOptimismUpgradeable;
  }
}
