// @openzeppelin/contracts-upgradeable/build/contracts/CrossChainEnabledArbitrumL1Upgradeable__factory.ts
/* Autogenerated file. Do not edit manually. */
/* tslint:disable */
/* eslint-disable */

import { Contract, Signer, utils } from "ethers";
import type { Provider } from "@ethersproject/providers";
import type {
  CrossChainEnabledArbitrumL1Upgradeable,
  CrossChainEnabledArbitrumL1UpgradeableInterface,
} from "../../../../../../node_modules/@openzeppelin/contracts-upgradeable/build/contracts/CrossChainEnabledArbitrumL1Upgradeable";

const _abi = [
  {
    anonymous: false,
    inputs: [
      {
        indexed: false,
        internalType: "uint8",
        name: "version",
        type: "uint8",
      },
    ],
    name: "Initialized",
    type: "event",
  },
];

export class CrossChainEnabledArbitrumL1Upgradeable__factory {
  static readonly abi = _abi;
  static createInterface(): CrossChainEnabledArbitrumL1UpgradeableInterface {
    return new utils.Interface(
      _abi
    ) as CrossChainEnabledArbitrumL1UpgradeableInterface;
  }
  static connect(
    address: string,
    signerOrProvider: Signer | Provider
  ): CrossChainEnabledArbitrumL1Upgradeable {
    return new Contract(
      address,
      _abi,
      signerOrProvider
    ) as CrossChainEnabledArbitrumL1Upgradeable;
  }
}
