// @openzeppelin/contracts-upgradeable/build/contracts/IERC1820ImplementerUpgradeable__factory.ts
/* Autogenerated file. Do not edit manually. */
/* tslint:disable */
/* eslint-disable */

import { Contract, Signer, utils } from "ethers";
import type { Provider } from "@ethersproject/providers";
import type {
  IERC1820ImplementerUpgradeable,
  IERC1820ImplementerUpgradeableInterface,
} from "../../../../../../node_modules/@openzeppelin/contracts-upgradeable/build/contracts/IERC1820ImplementerUpgradeable";

const _abi = [
  {
    inputs: [
      {
        internalType: "bytes32",
        name: "interfaceHash",
        type: "bytes32",
      },
      {
        internalType: "address",
        name: "account",
        type: "address",
      },
    ],
    name: "canImplementInterfaceForAddress",
    outputs: [
      {
        internalType: "bytes32",
        name: "",
        type: "bytes32",
      },
    ],
    stateMutability: "view",
    type: "function",
  },
];

export class IERC1820ImplementerUpgradeable__factory {
  static readonly abi = _abi;
  static createInterface(): IERC1820ImplementerUpgradeableInterface {
    return new utils.Interface(_abi) as IERC1820ImplementerUpgradeableInterface;
  }
  static connect(
    address: string,
    signerOrProvider: Signer | Provider
  ): IERC1820ImplementerUpgradeable {
    return new Contract(
      address,
      _abi,
      signerOrProvider
    ) as IERC1820ImplementerUpgradeable;
  }
}
