// @openzeppelin/contracts-upgradeable/build/contracts/IFxMessageProcessorUpgradeable__factory.ts
/* Autogenerated file. Do not edit manually. */
/* tslint:disable */
/* eslint-disable */

import { Contract, Signer, utils } from "ethers";
import type { Provider } from "@ethersproject/providers";
import type {
  IFxMessageProcessorUpgradeable,
  IFxMessageProcessorUpgradeableInterface,
} from "../../../../../../node_modules/@openzeppelin/contracts-upgradeable/build/contracts/IFxMessageProcessorUpgradeable";

const _abi = [
  {
    inputs: [
      {
        internalType: "uint256",
        name: "stateId",
        type: "uint256",
      },
      {
        internalType: "address",
        name: "rootMessageSender",
        type: "address",
      },
      {
        internalType: "bytes",
        name: "data",
        type: "bytes",
      },
    ],
    name: "processMessageFromRoot",
    outputs: [],
    stateMutability: "nonpayable",
    type: "function",
  },
];

export class IFxMessageProcessorUpgradeable__factory {
  static readonly abi = _abi;
  static createInterface(): IFxMessageProcessorUpgradeableInterface {
    return new utils.Interface(_abi) as IFxMessageProcessorUpgradeableInterface;
  }
  static connect(
    address: string,
    signerOrProvider: Signer | Provider
  ): IFxMessageProcessorUpgradeable {
    return new Contract(
      address,
      _abi,
      signerOrProvider
    ) as IFxMessageProcessorUpgradeable;
  }
}
